2025-06-19 11:47:06.411 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2025-06-19 11:47:06.919 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=24 from 2025-06-18 17:34:13.009682)
2025-06-19 11:47:28.869 WARNING (MainThread) [homeassistant.components.mqtt.sensor] The unit of measurement `ppm` is not valid together with device class `gas`. this will stop working in HA Core 2025.7.0
2025-06-19 11:47:28.876 WARNING (MainThread) [homeassistant.components.mqtt.sensor] The unit of measurement `%` is not valid together with device class `water`. this will stop working in HA Core 2025.7.0
2025-06-19 11:47:34.409 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Allumer Ventilo' trigger: In 'numeric_state' condition: unknown entity sensor.temperature_salon
2025-06-19 11:47:34.410 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Allumer Ventilo' trigger: In 'numeric_state' condition: unknown entity sensor.humidite_salon
2025-06-19 11:47:34.412 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Eteindre le ventilateur' trigger: In 'numeric_state' condition: unknown entity sensor.temperature_salon
2025-06-19 11:47:34.414 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Eteindre le ventilateur' trigger: In 'numeric_state' condition: unknown entity sensor.humidite_salon
2025-06-19 11:48:25.417 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.gaz_salon_2 (<class 'homeassistant.components.mqtt.sensor.MqttSensor'>) is using native unit of measurement 'ppm' which is not a valid unit for the device class ('gas') it is using; expected one of ['m続', 'ft続', 'CCF']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+mqtt%22
2025-06-19 11:48:25.448 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.niveau_citerne_eau (<class 'homeassistant.components.mqtt.sensor.MqttSensor'>) is using native unit of measurement '%' which is not a valid unit for the device class ('water') it is using; expected one of ['CCF', 'L', 'ft続', 'gal', 'm続']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+mqtt%22
2025-06-19 11:48:25.487 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:48:25.516 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:48:35.460 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:48:35.466 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:48:45.524 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:48:45.535 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:48:55.461 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:48:55.472 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:05.479 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:05.487 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:15.509 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:15.517 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:25.465 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:25.472 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:35.458 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:35.494 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:45.491 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:45.498 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:49:55.520 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:49:55.527 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:05.470 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:05.478 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:15.487 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:15.493 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:25.526 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:25.536 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:35.488 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:35.501 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:45.468 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:45.472 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:50:55.516 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:50:55.527 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:51:05.644 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:51:05.654 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:51:15.487 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:51:15.500 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:51:25.615 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:51:25.621 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:51:35.555 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:51:35.563 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:51:45.809 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:51:45.816 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:52:05.541 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:52:05.547 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:52:08.633 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:52:08.643 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:52:26.630 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:52:26.641 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: OFF
2025-06-19 11:52:35.529 WARNING (MainThread) [homeassistant.components.mqtt.alarm_control_panel] Received unexpected payload: DISARMED
2025-06-19 11:53:34.527 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:53:34.531 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:54:04.532 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:54:04.534 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:54:34.536 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:54:34.539 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:55:04.540 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:55:04.543 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:55:34.544 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:55:34.546 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:56:04.547 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:56:04.551 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:56:34.551 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:56:34.554 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:57:04.554 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:57:04.558 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
2025-06-19 11:57:15.760 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=26 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=26, family=10, type=2, proto=0, laddr=('::', 39854, 0, 0)>
2025-06-19 11:57:15.762 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=27 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=27, family=2, type=2, proto=0, laddr=('0.0.0.0', 38811)>
2025-06-19 11:57:15.763 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=27 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=27, family=2, type=2, proto=0, laddr=('0.0.0.0', 38811)>
2025-06-19 11:57:34.558 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=41 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=41, family=2, type=2, proto=0, laddr=('0.0.0.0', 44776)>
2025-06-19 11:57:34.562 ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=46 read=polling write=<idle, bufsize=0>>, socket: <asyncio.TransportSocket fd=46, family=10, type=2, proto=0, laddr=('::', 59428, 0, 0)>
